WordPress/Classes/ViewRelated/Post/PostSettingsViewController+JetpackSocial.swift

```diff
@@ -25,4 +32,10 @@ extension PostSettingsViewController {
         return viewController.view
     }
 
+    private func availableServices() -> [PublicizeService] {
+        let context = apost.managedObjectContext ?? ContextManager.shared.mainContext
+        let services = try? PublicizeService.allPublicizeServices(in: context)
+        return  services ?? []
```

Comment:
Nit: there's an extra space here.
```suggestion
        return services ?? []
```

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/20823/commits/986811cde481460b1ba3c7189e18b63875737c4d, thanks for the catch!